kDrive/UI/Controller/Menu/ParameterTableViewController.swift

```diff
@@ -40,35 +57,55 @@ class ParameterTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return 5
+        return tableContent.count
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        if indexPath.row >= 0 && indexPath.row <= 2 {
+        switch tableContent[indexPath.row] {
+        case .photos:
+            let cell = tableView.dequeueReusableCell(type: ParameterTableViewCell.self, for: indexPath)
+            cell.initWithPositionAndShadow(isFirst: true)
+            cell.titleLabel.text = KDriveStrings.Localizable.syncSettingsTitle
+            cell.valueLabel.text = PhotoLibraryUploader.instance.isSyncEnabled ? KDriveStrings.Localizable.allActivated : KDriveStrings.Localizable.allDisabled
+            return cell
+        case .theme:
             let cell = tableView.dequeueReusableCell(type: ParameterTableViewCell.self, for: indexPath)
-            if indexPath.row == 0 {
-                cell.initWithPositionAndShadow(isFirst: true)
-                cell.titleLabel.text = KDriveStrings.Localizable.syncSettingsTitle
-                cell.valueLabel.text = PhotoLibraryUploader.instance.isSyncEnabled ? KDriveStrings.Localizable.allActivated : KDriveStrings.Localizable.allDisabled
-            } else if indexPath.row == 1 {
-                cell.initWithPositionAndShadow()
-                cell.titleLabel.text = KDriveStrings.Localizable.notificationTitle
-                cell.valueLabel.text = KDriveStrings.Localizable.notificationAll
-            } else if indexPath.row == 2 {
-                cell.initWithPositionAndShadow()
-                cell.titleLabel.text = KDriveStrings.Localizable.appSecurityTitle
-                cell.valueLabel.text = UserDefaults.shared.isAppLockEnabled ? KDriveStrings.Localizable.allActivated : KDriveStrings.Localizable.allDisabled
+            cell.initWithPositionAndShadow()
+            cell.titleLabel.text = KDriveStrings.Localizable.settingDarkmodeTitle
+            let defaults = UserDefaults.standard
+            let theme = defaults.object(forKey: "Theme") as! String
```

Comment:
When you add the computed property to `UserDefaults` you will be able to use `UserDefaults.shared.theme`